src/test/java/co/rsk/bitcoinj/script/P2ShErpRedeemScriptParserTest.java

```diff
@@ -40,30 +57,29 @@ public void extractStandardRedeemScriptChunks_fromStandardRedeemScript_fail() {
             defaultRedeemScriptKeys
         );
 
-        P2shErpFederationRedeemScriptParser.extractStandardRedeemScriptChunks(standardRedeemScript.getChunks());
+        new P2shErpRedeemScriptParser(standardRedeemScript.getChunks());
     }
 
     @Test
-    public void createP2shErpRedeemScript() {
+    public void createP2shErpRedeemScript_whenValidInputs_shouldCreateRedeemScript() {
         Script defaultFederationRedeemScript = RedeemScriptUtils.createStandardRedeemScript(
             defaultRedeemScriptKeys
         );
         Script erpFederationRedeemScript = RedeemScriptUtils.createStandardRedeemScript(
             emergencyRedeemScriptKeys
         );
-        Long csvValue = 300L;
 
-        Script erpRedeemScript = P2shErpFederationRedeemScriptParser.createP2shErpRedeemScript(
+        Script erpRedeemScript = P2shErpRedeemScriptParser.createP2shErpRedeemScript(
```

Comment:
We should avoid using this `create` method since it'll be removed. We might use the methods from `RedeemScriptUtils`. WDYT?